client/components/room/Workspace.jsx

```diff
@@ -1,14 +1,46 @@
 import React from 'react';
+import CodeMirror from 'react-codemirror';
+
+require('codemirror/lib/codemirror.css');
+require('codemirror/mode/javascript/javascript');
+require('codemirror/mode/xml/xml');
+require('codemirror/mode/markdown/markdown');
+require('codemirror/mode/python/python');
+require('codemirror/mode/ruby/ruby');
 
 class Workspace extends React.Component {
   constructor(props) {
     super(props);
+    this.state = {
+      code: '// Ribbit'
+    };
+  }
+
+  getInitialState() {
+    return {
+      code: '// Code'
+    };
+  }
+
+  updateCode(newCode) {
```

Comment:
To prevent the errors on the app that come up, just bind updateCode to 'this' in the constructor :)